packages/lesswrong/components/users/DialogueSuggestionsPage.tsx

```diff
@@ -0,0 +1,782 @@
+  const currentUser = useCurrentUser();
+
+  const {loading: userLoading, results : userDialogueChecks} = useMulti({
+    terms: {
+      view: "userDialogueChecks",
+      userId: currentUser?._id,
+      limit: 1000,
+    },
+    fragmentName: "DialogueCheckInfo",
+    collectionName: "DialogueChecks",
+  });
+
+  const [localDialogueChecks, setLocalDialogueChecks] = useState(userDialogueChecks ?? []);
+
+  const {loading: userOptedInLoading, results : UsersOptedInToDialogueFacilitation, loadMoreProps} = useMulti({
+    terms: { 
+      view: 'usersWithOptedInToDialogueFacilitation',
+      limit: 10, 
+    },
+    fragmentName: 'UsersOptedInToDialogueFacilitation',
+    collectionName: 'Users'  
+  });
+
+  // // get all check rows where user is currentUser and checked is true
+  // const GET_USERS_DIALOGUE_CHECKS = gql`
+  //   query getUsersDialogueChecks {
+  //     getUsersDialogueChecks {
+  //       _id
+  //       __typename
+  //       userId
+  //       targetUserId
+  //       checked
+  //       match
+  //     }
+  //   }
+  // `;
+
+  // // useMulti fetching from dialogueChecks based on a view, which takes as a usedId as an input. 
+  // // permission stuff happens in the background
+  
+  // const { loading: loadingChecks, error: errorChecks, data: dataChecks } = useQuery(GET_USERS_DIALOGUE_CHECKS);  // for all the targetUsers thus obtained, check if there's a match 
+
+  let targetUserIds = [];
+  if (userDialogueChecks) {
+    targetUserIds = userDialogueChecks.map(check => check.targetUserId);
+  }
+
+  async function createDialogue(title: string, participants: string[]) {
+    const createResult = await createPost({
+      data: {
+        title,
+        draft: true,
+        collabEditorDialogue: true,
+        coauthorStatuses: participants.map(userId => ({userId, confirmed: true, requested: false})),
+        shareWithUsers: participants,
+        sharingSettings: {
+          anyoneWithLinkCan: "none",
+          explicitlySharedUsersCan: "edit",
+        },
+        contents: {
+          originalContents: {
+            type: "ckEditorMarkup",
+            data: ""
+          }
+        } as AnyBecauseHard
+      },
+    });
+    if (createResult?.data?.createPost?.data) {
+      const post = createResult?.data?.createPost?.data;
+      if (post) {
+        const postId = post._id;
+        const postEditUrl = `/editPost?postId=${postId}`;
+        history.push(postEditUrl);
+      }
+    }
+  }
+
+  if (!currentUser) return <p>You have to be logged in to view this page</p>
+  if (loading) return <p>Loading...</p>;
+  if (error || !userDialogueChecks || userDialogueChecks.length > 1000) return <p>Error </p>; // if the user has clicked that much stuff things might break...... 
+  if (userDialogueChecks?.length > 1000) {
+    console.log("Warning: userDialogueChecks.length > 1000, seems user has checked more than a thousand boxes? how is that even possible? let a dev know and we'll fix it...")
+  }
```

Comment:
Mm... if we don't want users checking a thousand checkboxes, there are probably better ways to handle this.